packages/nodes-base/nodes/GraphQL/GraphQL.node.ts

```diff
@@ -418,40 +418,45 @@ export class GraphQL implements INodeType {
 
 				const gqlQuery = this.getNodeParameter('query', itemIndex, '') as string;
 				if (requestMethod === 'GET') {
-					if (!requestOptions.qs) {
-						requestOptions.qs = {};
-					}
+					requestOptions.qs = requestOptions.qs ?? {};
 					requestOptions.qs.query = gqlQuery;
-				} else {
-					if (requestFormat === 'json') {
-						const jsonBody = {
-							...requestOptions.body,
-							query: gqlQuery,
-							variables: this.getNodeParameter('variables', itemIndex, {}) as object,
-							operationName: this.getNodeParameter('operationName', itemIndex) as string,
-						};
-						if (typeof jsonBody.variables === 'string') {
-							try {
-								jsonBody.variables = JSON.parse(jsonBody.variables || '{}');
-							} catch (error) {
-								throw new NodeOperationError(
-									this.getNode(),
-									'Using variables failed:\n' +
-										(jsonBody.variables as string) +
-										'\n\nWith error message:\n' +
-										(error as string),
-									{ itemIndex },
-								);
-							}
-						}
-						if (jsonBody.operationName === '') {
-							jsonBody.operationName = null;
```

Comment:
I think this behavior was important. With the Linear GraphQL API for example this throws an error now.
You can use the node from the ticket description to reproduce it.